modules/administration-guide/partials/proc_configuring-prod-cli-with-environment-variables-to-use-an-sftp-backup-server.adoc

```diff
@@ -0,0 +1,19 @@
+[id="configuring-prod-cli-for-an-sftp-backup-server-with-environment-variables_{context}"]
+= Configuring {prod-cli} for an SFTP backup server with environment variables
+
+As an alternative to using command-line options for `{prod-cli} server:backup`, you can configure {prod-cli} for an SFTP backup server with environment variables that will allow you to run `{prod-cli} server:backup` without having to enter those arguments as options.
+
+.Prerequisites
+
+* Installed `{prod-cli}`.
+* Configured SFTP backup server. See xref:supported-restic-compatible-backup-servers.adoc[].
+
+.Procedure
+
+* Set the following environment variables:
+** `BACKUP_REPOSITORY_URL` for the backup repository.
+** `BACKUP_REPOSITORY_PASSWORD` for the backup repository password.
+** To log in without a password, set the SSH key using either environment variable:
```

Comment:
This line stands out from the list of variables, maybe it could be worked around somehow?

For example, the previous line could begin with "To log in with a password, set `BACKUP_REPOSITORY_PASSWORD`..." It would logically unite lines 15 and 16 and show that one excludes the other.

Or this point on line 16 could become a subpoint of BACKUP_REPOSITORY_PASSWORD since they're parallel procedures, can't be done at once.

Reply:
@missmesss, great suggestion. This is how I rewrote it:

![image](https://user-images.githubusercontent.com/78564304/147359948-bcfc41f9-dbcf-4330-8d2a-bc8ab1223752.png)